src/open_inwoner/configurations/admin.py

```diff
@@ -29,13 +29,14 @@
 @admin.action(description=_("Delete selected websites"))
 def delete_selected(modeladmin, request, queryset):
     """
-    Override `delete_selected` action to prevent accidental deletion of all websites
+    Override `delete_selected` action to prevent accidental deletion of the primary website
     """
-    if queryset.count() == Site.objects.all().count():
+    primary_site = Site.objects.get(id=1)
```

Comment:
I'd recommend to use settings.SITE_ID instead of hardcoding this

Reply:
Good call, though now I've revamped the code completely. If we have no use for more than one site, we can simply block adding more sites and then block deleting altogether (at least via the admin), so there's no need to fish around for the site id.